_posts/2018-02-05-lets-step-in-paris.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title: "How about a french adventure in Paris?"
+socialImageSrc: "/assets/paris/paris-panorama.jpg"
+authors:
+  - GuillaumeGomez
+---
+
+How about enjoying Rust in the french capital? After the adventures in Berlin, Київ and Zürich, it's time to put our luggages in Paris, France! We hope you'll join us on **May 26th and October 27th, 2018** at the incredible [**Espace Saint-Martin**](http://espacesaintmartin.com/fr/contact/) in the center of Paris.
+
+[![Paris the city](assets/paris/paris-panorama.jpg)](https://www.flickr.com/photos/)
```

Comment:
We need two pictures to put in here (a small one and a big one).

Reply:
Do you have any nice shots? Cell phone shots are totally okay.

Reply:
Funny thing: I've been living in paris for years and I have no outside pics. But I think it's pretty easy to find considering it's Paris...